liteflow-labs/starter-kit
# Webhook email notification + bid created and auction bid created

### Project organization

- Dependency: https://github.com/liteflow-labs/starter-kit/pull/140

### Description

Add a new `/api/notification` function that can be added as a custom webhook.
This also adds new email support with https://react.email/ that are sent directly from this function.
I also added 2 emails
- `bid created` when a bid is created
- `auction bid created` when a bid is created from an auction

### How to test

Setup new required envs
- EMAIL_PASSWORD
- EMAIL_FROM
- LITEFLOW_WEBHOOK_SECRET
Use the new webhook feature on the infra and connect the following webhooks
- `BID_CREATED` to the `/api/notification`
- `AUCTION_BID_CREATED` to the `/api/notification`
Create a new open bid or auction bid and check that you received your email correctly
